r2/r2/templates/subredditstylesheetsource.html

```diff
@@ -33,3 +33,31 @@
 </pre>
 
 ${unsafe(js.use("highlight"))}
+
+%if len(thing.images) > 0:
+  <div id="images">
+    <h2><a name="images">${_("images")}</a></h2>
+    <ul id="image-preview-list" class="image-list">
+      <%def name="make_li(name='', img=None, prototype=False)">
+        <li ${"style='display:none'" if img is None else ""}>
+          <a href="${img}" class="preview">
+            <img id="img-preview-${name}" src="${img}" 
+                 alt="Image ${name}" title="click to preview"/>
+          </a>
+          <div class="description">
+            <b class="img-name">
+              ${name}
+            </b>
+            <br/>
+            <span>link:</span>
+            <pre class="img-url">url(%%${name}%%)</pre>
+            <br/>
+          </div>
+        </li>
+        </%def>
+      ${make_li(prototype=True)}
```

Comment:
I don't understand why this is here, as it adds a empty image both here and in subredditstylesheet.html, which seems unnecessary. Just in case I left it in both,